languages/julia/exercises/concept/unicode-identifiers/.meta/example.jl

```diff
@@ -0,0 +1,29 @@
+using REPL.REPLCompletions: emoji_symbols
+
+import Base: +, -, show
+
+struct Clock
+    t::Rational{Int}
+
+    function Clock(t::Rational{Int})
+        t %= 12
+        new(t < 0 ? t + 12 : t)
+    end
+end
+
++(c₁::Clock, c₂::Clock) = Clock(c₁.t + c₂.t)
+-(c₁::Clock, c₂::Clock) = Clock(c₁.t - c₂.t)
+
+# function show(io::IO, c::Clock)
+#     if c.t % 2 == 0
+#         show(io, emoji_symbols["\\:clock$(Int(c.t)):"])
+#     else
+#         t = Int(c.t - 1//2)
+#         show(io, emoji_symbols["\\:clock$(t)30:"])
+#     end
+# end
+
+for i in 1:12
+    @eval const $(Symbol(emoji_symbols["\\:clock$i:"])) = Clock($i//1)
+    @eval const $(Symbol(emoji_symbols["\\:clock$(i)30:"])) = Clock($i + 1//2)
+end
```

Comment:
Writing them out also works, so I didn't add prerequisite concepts for `@eval`/metaprogramming.